lale/expressions.py

```diff
@@ -242,8 +242,10 @@ def _make_binop(op, left, other):
         return False
 
 
-def _make_ast_expr(arg: Union[Expr, int, float, str, AstExpr]) -> AstExpr:
-    if isinstance(arg, Expr):
+def _make_ast_expr(arg: Union[None, Expr, int, float, str, AstExpr]) -> AstExpr:
+    if arg is None:
+        return ast.Constant(value=None)
```

Comment:
I will be curious to see if it works because it looks like Python 3.7 and 3.8 have different representation of `None`. See for example:
```
Python 3.7.12 (default, Dec  6 2021, 22:38:59) 
[Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ast
>>> ast.dump(ast.parse("None"))
'Module(body=[Expr(value=NameConstant(value=None))])'
```
and
```
Python 3.8.12 (default, Nov 16 2021, 15:11:11) 
[Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ast
>>> ast.dump(ast.parse("None"))
'Module(body=[Expr(value=Constant(value=None, kind=None))], type_ignores=[])'
```

Reply:
I didn't know Python 3.7 and 3.8 had different representations of `None`. Fortunately, the regression tests passed on both Python 3.7 and 3.8, so I guess the representation in the code now works for both.